waspc/examples/todoApp/src/client/pages/auth/appearance.js

```diff
@@ -0,0 +1,10 @@
+
+const appearance = {
+  colors: {
+    brand: '#5969b8', // blue
```

Comment:
Instead of writing the colors in comments, you could do two tokens like where one refers to the other by name:
```
blue: '#5969b8',
brand: '$blue',
```

Reply:
this is in example app - with this, we're exposing that we are internally using Stitches, do we want that?

Reply:
It's a nice practice and we would give users an example how to organise their code.